<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_14" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="btn_back">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../img/icons8-back-26.png</normaloff>../img/icons8-back-26.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignVCenter">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../img/icons8-animal-shelter-50.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../img/icons8-multiply-26.png</normaloff>../img/icons8-multiply-26.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_15" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">color: #1b1209;
font: 18pt &quot;Bahnschrift&quot;;</string>
           </property>
           <property name="text">
            <string>What type of pet are you searching for?</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_16" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="widget" native="true">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
	background-color: white;
	border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    border-color: #ECE7E0;
}
QWidget:hover{
background-color: #C3BBAB;
border-radius: 15px;
}
</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="btn_rabbit">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true"> border-color: rgba(255, 0, 0, 0);
background-color: rgba(255, 0, 0, 0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../img/icons8-rabbit-50.png</normaloff>../img/icons8-rabbit-50.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_rabbitName">
              <property name="styleSheet">
               <string notr="true">border-color: rgba(255, 0, 0, 0);
color: #356a9c;
font: 600 14pt &quot;Bahnschrift&quot;;
background-color: rgba(255, 0, 0, 0);</string>
              </property>
              <property name="text">
               <string>Rabbit</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QWidget" name="widget_2" native="true">
           <property name="styleSheet">
            <string notr="true">QWidget{
	background-color: white;
	border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    border-color: #ECE7E0;
}
QWidget:hover{
background-color: #C3BBAB;
border-radius: 15px;
}
</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="btn_hamster">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true"> border-color: rgba(255, 0, 0, 0);
background-color: rgba(255, 0, 0, 0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../img/icons8-hamster-50.png</normaloff>../img/icons8-hamster-50.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_hamsterName">
              <property name="styleSheet">
               <string notr="true">border-color: rgba(255, 0, 0, 0);
color: #356a9c;
font: 600 14pt &quot;Bahnschrift&quot;;
background-color: rgba(255, 0, 0, 0);</string>
              </property>
              <property name="text">
               <string>Hamster</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QWidget" name="widget_3" native="true">
           <property name="styleSheet">
            <string notr="true">QWidget{
	background-color: white;
	border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    border-color: #ECE7E0;
}
QWidget:hover{
background-color: #C3BBAB;
border-radius: 15px;
}
</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="btn_bird">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true"> border-color: rgba(255, 0, 0, 0);
background-color: rgba(255, 0, 0, 0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../img/icons8-bird-50.png</normaloff>../img/icons8-bird-50.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_birdName">
              <property name="styleSheet">
               <string notr="true">border-color: rgba(255, 0, 0, 0);
color: #356a9c;
font: 600 14pt &quot;Bahnschrift&quot;;
background-color: rgba(255, 0, 0, 0);</string>
              </property>
              <property name="text">
               <string>Bird</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QWidget" name="widget_4" native="true">
           <property name="styleSheet">
            <string notr="true">QWidget{
	background-color: white;
	border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    border-color: #ECE7E0;
}
QWidget:hover{
background-color: #C3BBAB;
border-radius: 15px;
}
</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="btn_pig">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true"> border-color: rgba(255, 0, 0, 0);
background-color: rgba(255, 0, 0, 0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../img/icons8-pig-50.png</normaloff>../img/icons8-pig-50.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_pigName">
              <property name="styleSheet">
               <string notr="true">border-color: rgba(255, 0, 0, 0);
color: #356a9c;
font: 600 14pt &quot;Bahnschrift&quot;;
background-color: rgba(255, 0, 0, 0);</string>
              </property>
              <property name="text">
               <string>Pig</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
